igvf_dump_metadata.py

```diff
@@ -0,0 +1,288 @@
+import argparse
+import os
+import requests
+
+import pandas as pd
+from collections import defaultdict
+
+# prerequisite libraries:
+# pip install pandas
+# pip install openpyxl
+
+# You will need to put API credentials in the environment variables: IGVF_API_KEY & IGVF_SECRET_KEY.
+
+parser = argparse.ArgumentParser(
+    prog='IGVF Metadata Table',
+    description='Generates meatadata table for a given analysis set'
+)
+
+parser.add_argument('-a', '--accession', help='accession of one analysis set')
+
+# will add this if useful
+# parser.add_argument('-i', '--infile', help='A file containing a list of measurement set accessions.')
+
+# api connection setting from environment variables.
+# could add alternative ways
+if os.getenv('IGVF_API_KEY', None) and os.getenv('IGVF_SECRET_KEY', None):
+    auth = (os.getenv('IGVF_API_KEY'), os.getenv('IGVF_SECRET_KEY'))
+
+url = 'https://api.data.igvf.org/'
+
+# set the properties/columns here:
+# any column with all rows = None will be dropped in final tables
+basic_props = ['@id', 'accession', 'aliases', 'status', 'audit']
+link_obj_props = {'input_file_sets': ['measurement_sets', 'auxiliary-sets', 'samples', 'donors', 'library_construction_platform', 'assay_term', 'documents'],
```

Comment:
I think it makes sense to add 'dbxrefs', 'protocols', 'control_file_sets', 'control_for', 'related_multiome_datasets', 'multiome_size' to check for consistency between these properties on the input file sets.

Reply:
'dbxrefs', 'protocols', 'multiome_size' are not links but should be added to output_props I guess